core/stream/pipeline.pl

```diff
@@ -210,6 +216,15 @@ (;$)
   }
 }
 
+use constant excel_cat => gen pydent q{
```

Comment:
@spencertipping there's a bug here i'm not sure how to fix; `pydent` isn't imported/recognized here, even though it is in `core/matrix/matrix.pl`. I assume it's something to do with the order of `ni` compilation options that I don't understand

Reply:
Yep, that's exactly what's happening here. `matrix.pl` depends on a bunch of stream stuff, so we include the stream module first.

A couple of fixes are to manually outdent (which I did for now), or to migrate the `pydent` function into `core/util` or something so it's available everywhere. I'd be cool with either one.